atmosphere/demo/demo.h

```diff
@@ -87,6 +90,8 @@ class Demo {
   bool show_help_;
 
   std::unique_ptr<Model> model_;
+  GLuint vao_;
+  GLuint vbo_;
```

Comment:
Please use more explicit names, for instance fullscreen_quad_vao_ (https://google.github.io/styleguide/cppguide.html#General_Naming_Rules).